File: Dental_Clinic_BE/material-service/src/main/java/com/dental_clinic/material_service/DTO/Response/MedicineRes.java
package com.dental_clinic.material_service.DTO.Response;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.experimental.FieldDefaults;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

@Getter
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class MedicineRes {
    Long id;
    String name;
    Integer quantity;
    String unit;
    LocalDate mfg_date;
    boolean able;
    Integer price;
    Integer cost;
    String instruction;
    String cared_actor;
    List<Long> ingreIdList;
    String func;
    Long categoryId;
    String img;
    LocalDateTime created_at;

}

File: Dental_Clinic_FE/dental_clinic/src/app/share/dto/response/medicine-res.ts
export interface MedicineRes {
    id: number;
    name: string;
    quantity: number;
    unit: string;
    mfg_date: string;
    able: boolean;
    price: number;
    cost: number;
    func: string;
    ingreIdList: number[];
    instruction: string;
    cared_actor: string;
    img: string;
    categoryId: number;
}